.travis.yml

```diff
@@ -3,9 +3,12 @@ language: node_js
 node_js:
   - "6"
 
+before_install:
+  - npm install yarn -g
```

Comment:
it seems that yarn is already available without the need to install anything. `.travis.yml` script works without the need to specicy `before_install` and `install` steps. I have added `cache` as suggested:

```yml
cache:
  yarn: true
  directories:
    - node_modules
```